sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRepositoryAsync.java

```diff
@@ -348,8 +350,8 @@ public RegistryArtifactAsync getArtifact(String digest) {
                 .setDeleteEnabled(repositoryProperties.isDeleteEnabled())
                 .setListEnabled(repositoryProperties.isListEnabled())
                 .setWriteEnabled(repositoryProperties.isWriteEnabled())
-                .setReadEnabled(repositoryProperties.isReadEnabled())
-                .setTeleportEnabled(repositoryProperties.isTeleportEnabled());
+                .setReadEnabled(repositoryProperties.isReadEnabled());
+//                .setTeleportEnabled(repositoryProperties.isTeleportEnabled());
```

Comment:
Was this property removed?

Reply:
Yes, the service decided to remove it for this api-version. hence, I chose to comment rather than completely remove.